commons/src/main/res/values-da/strings.xml

```diff
@@ -281,19 +281,19 @@
     <string name="subsampling_title">Subsampling Scale Image View (zoomable imageviews)</string>
     <string name="glide_title">Glide (image loading and caching)</string>
     <string name="picasso_title">Picasso (image loading and caching)</string>
-    <string name="cropper_title">Android Image Cropper (image crop and rotate)</string>
-    <string name="multiselect_title">RecyclerView MultiSelect (selecting multiple list items)</string>
-    <string name="rtl_viewpager_title">RtlViewPager (right to left swiping)</string>
-    <string name="joda_title">Joda-Time (Java date replacement)</string>
+    <string name="cropper_title">Android Image Cropper (beskær og roter billede)</string>
+    <string name="multiselect_title">RecyclerView MultiSelect (vælger flere elementer)</string>
+    <string name="rtl_viewpager_title">RtlViewPager (højre til venstre-swiping)</string>
+    <string name="joda_title">Joda-Time (Java-dato-erstatning)</string>
     <string name="stetho_title">Stetho (debugging databases)</string>
     <string name="otto_title">Otto (event bus)</string>
-    <string name="photoview_title">PhotoView (zoomable GIFs)</string>
-    <string name="pattern_title">PatternLockView (pattern protection)</string>
-    <string name="reprint_title">Reprint (fingerprint protection)</string>
-    <string name="gif_drawable_title">Gif Drawable (loading GIFs)</string>
+    <string name="photoview_title">PhotoView (zoombare GIF'er)</string>
```

Comment:
@10cents dont forget escaping apostrophes with a backslash in the future (I fixed these already), replacing GIF'er with GIF\'er. Thanks

Reply:
Oh no, sorry, sorry. I'll do my best!